src/compiler/commandLineParser.ts

```diff
@@ -71,14 +71,15 @@ namespace ts {
         ["es2021.promise", "lib.es2021.promise.d.ts"],
         ["es2021.string", "lib.es2021.string.d.ts"],
         ["es2021.weakref", "lib.es2021.weakref.d.ts"],
-        ["esnext.array", "lib.es2019.array.d.ts"],
+        ["es2021.string", "lib.es2021.string.d.ts"],

```

Comment:
`es2021.string` is already present on line 72.

```suggestion
```

Reply:
The duplication exists so that anyone who previously used `--lib esnext.string` or `--lib esnext.array` and expected to get `String.prototype.replaceAll` or `Array.prototype.flatMap` would not be broken.